instance/models/instance.py

```diff
@@ -547,6 +685,80 @@ def studio_url(self):
         """
         return u'{0.protocol}://{0.studio_domain}/'.format(self)
 
+    @property
+    def database_name(self):
+        """
+        The database name used for external databases. Escape all non-ascii characters and truncate to 64 chars, the
+        maximum for mysql
+        """
+        name = self.domain.replace('.', '_')
+        allowed = string.ascii_letters + string.digits + '_'
+        escaped = ''.join(char for char in name if char in allowed)
+        return escaped[:64]
```

Comment:
That's a useful function, I didn't know about it. I have made the change, although I think collisions here are pretty unlikely in practice.